index.js

```diff
@@ -95,6 +99,15 @@ class Ky {
 			retry: 2,
 			...otherOptions
 		};
+		let abortController;
+
+		if (supportsAbortController) {
+			abortController = new AbortController();
+			if (this._options.signal) {
+				this._options.signal.addEventListener('abort', () => abortController.abort());
+			}
+			this._options.signal = abortController.signal;
```

Comment:
Good. First I thought it should be in the `else` statement but then it would make no sense. 👍